components/src/core/components/Button/Button.vue

```diff
@@ -121,6 +122,12 @@ export default defineComponent({
       };
     },
   },
+
+  methods: {
+    mouseDown(e: Event) {
+      e.preventDefault();
```

Comment:
Why do you need to do this? @Super-Chama any idea about this?

Reply:
this is a different event than click, that occur before click event. I'm okay to go with click.

Reply:
if going with click, don't use `preventDefault()` or `@click.prevent`

Reply:
This change has removed with commit b780fdf71c0182b778691e846efe59b33da9f740